client-common/src/main/java/org/apache/cassandra/sidecar/common/request/ListOperationsJobsRequest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.request;
+
+import io.netty.handler.codec.http.HttpMethod;
+import org.apache.cassandra.sidecar.common.ApiEndpointsV1;
+import org.apache.cassandra.sidecar.common.response.ListOperationsJobsResponse;
+
+/**
+ * Represents a request to retrieve the list of async operations jobs
+ */
+public class ListOperationsJobsRequest extends JsonRequest<ListOperationsJobsResponse>
```

Comment:
I don't have a strong opinion here. The reason I did not choose "Operational" is because it seems to suggest something is "working" or "in-use" and not specifically jobs that are used in Operations context (which seemed more appropriate). Lmk what you think.

Reply:
"Operational" also describes something that is related to operations.

> of, relating to, or based on operations

See https://www.merriam-webster.com/dictionary/operational

It is confusing to see two nouns putting together. Looks like Hari has the same confusion. 

Please update the name for clarity.

Reply:
sounds reasonable. "Operational" it is.